jfix-stdlib-ratelimiter/src/test/kotlin/ru/fix/stdlib/ratelimiter/TestCommonPoolScope.kt

```diff
@@ -0,0 +1,20 @@
+package ru.fix.stdlib.ratelimiter
+
+import kotlinx.coroutines.CoroutineExceptionHandler
+import kotlinx.coroutines.CoroutineName
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.asCoroutineDispatcher
+import org.slf4j.LoggerFactory
+import java.util.concurrent.ForkJoinPool
+import kotlin.coroutines.EmptyCoroutineContext
+
+object TestCommonPoolScope : CoroutineScope {
```

Comment:
Is it really necessary to make custom CoroutineScope based on ForkJoinPool? Why don't you want to use standart `Dispatchers.Default` ?

Reply:
Ok, TestCommonPoolScope removed